packages/vscode-extension/src/commands/DeployValidateCommon.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ ** Copyright (c) 2021 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+import { window, QuickPickItem } from 'vscode';
+import { COMMAND, DEPLOY, VALIDATE, ANSWERS } from '../service/TranslationKeys';
+import { ApplicationConstants, ProjectInfoService, actionResultStatus } from '../util/ExtensionUtil';
+import BaseAction from './BaseAction';
+import { EOL } from 'os';
+
+const COMMAND_OPTIONS = {
+	OPTIONS: {
+		ACCOUNT_SPECIFIC_VALUES: 'accountspecificvalues',
+	},
+	FLAGS: {
+		APPLY_INSTALLATION_PREFS: 'applyinstallprefs',
+	},
+};
+
+const ACCOUNT_SPECIFIC_VALUES = {
+	ERROR: 'ERROR',
+	WARNING: 'WARNING',
+};
+
+interface commandOption extends QuickPickItem {
+	value: string;
+}
+
+export default abstract class DeployValidateCommon extends BaseAction {
```

Comment:
What do you think of renaming this to something like BaseDeployValidateAction or DeployValidateCommonAction?

Reply:
As discussed, I'm good with DeployValidateCommon. No changes needed.